### PR TITLE
Fixed issue #30251   Html tag <br> visible in message

### LINKED ISSUE #30251: Html tag <br> visible in message
**Precondition:**

Install Magento 2.4.1

Create a simple product

Go to System->Export;

Select Entity TypeProducts and click "Continue";

Run php bin/magento cron:run;

Download the created export file;

**Step to Reproduced:**

Open the file with exported product and delete it's SKU. Save it

Go to **System -> Import**

Fill required fields and select exported file to import.
* **Entity Type:** Products;
* **Import Behavior:** Add/Update

Click on **Check Data**

**(/) Expected Result:**

Html tag **<br>** not visible

**(x) Actual Result:**

Html tag **<br>** visible in message

image

### PR DESCRIPTION
**Pull request by irshadcedcoss**
### Description (*)
Html tag <br> visible in message #30251

### Fixed Issues (if relevant)

1. Fixes magento/magento2#30251

### Manual testing scenarios (*)
Open the file with exported product and delete it's SKU. Save it

Go to System -> Import

Fill required fields and select exported file to import.

    Entity Type: Products;
    Import Behavior: Add/Update

Click on Check Data

(/) Expected Result:

Html tag
not visible

(x) Actual Result:

Html tag
 visible in message

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by irshadcedcoss** [comment]
#dmcdindia2020

File: app/code/Magento/ImportExport/Controller/Adminhtml/ImportResult.php [hunk 1]
```diff
@@ -68,7 +68,7 @@ protected function addErrorMessages(
             $message = '';
             $counter = 0;
             foreach ($this->getErrorMessages($errorAggregator) as $error) {
-                $message .= ++$counter . '. ' . $error . '<br>';
+                $message .= ++$counter . '. ' . $error . '</br>';
```
Comment (rogyar) [inline app/code/Magento/ImportExport/Controller/Adminhtml/ImportResult.php:71]: Hi @irshadcedcoss. Thank you for your contribution. Could you explain the purpose of the `</br>` tag, please?
There are two common versions of the br tag: `<br>` (HTML), `<br />` (XHTML). But not sure about your version.

Thanks.

**Review by rogyar — CHANGES_REQUESTED** [review]

**Comment by engcom-Charlie** [comment]
@irshadcedcoss please sing CLA and look at https://github.com/magento/magento2/pull/30415#discussion_r502789072. Otherwise, we can't proceed with your PR.
Thank you.

**Comment by engcom-Charlie** [comment]
@irshadcedcoss please look at https://github.com/magento/magento2/pull/30415#discussion_r502789072 https://github.com/magento/magento2/pull/30415#issuecomment-707574199. Otherwise, we can't proceed with your PR.
Thank you.

**Comment by engcom-Charlie** [comment]
Hi @irshadcedcoss, I'm closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for your collaboration.

**Pull request closed without merging**